tests/helpers.py

```diff
@@ -232,25 +231,6 @@ def longest_regex_prefix_match(regex, string, flags=0):
     return match
 
 
-def gcs_fs_wipe_path(url):
-    assert "BNTESTDATA" in url
```

Comment:
I like that we're deleting these methods, but it's a bit a shame to be losing these asserts. Should we be worried about the risk of deleting the wrong files on GCS?

Reply:
Left out by accident. `BNTESTDATA` check has been added back.